src/nginxconfig/generators/conf/website.conf.js

```diff
@@ -369,5 +369,18 @@ export default (domain, domains, global) => {
         }
     }
 
+    // Security.txt
+    if (global.security.securityTxt.computed) {
+        config.push(['# security.txt', '']);
+        config.push(['location /security.txt', {
+            return: `301 http://${domain.server.wwwSubdomain.computed ? 'www.' : ''}${domain.server.domain.computed}/.well-known/security.txt`,
+        }]);
+        
+        // Custom security.txt path
+        config.push(['location = /.well-known/security.txt', {
+            alias: `${global.security.securityTxtPath.value}`,
+        }]);
+    }
```

Comment:
This should be generated in security.conf.

We can remove the `http://${domain.server.wwwSubdomain.computed ? 'www.' : ''}${domain.server.domain.computed}`, and just redirect to the absolute URI `/.well-known/security.txt` -- this'll let nginx add the scheme/host for each domain as it sees fit.

Reply:
I've changed it. You can check it